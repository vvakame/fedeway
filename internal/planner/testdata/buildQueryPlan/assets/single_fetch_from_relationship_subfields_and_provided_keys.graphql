# schema: supergraphSdl.graphqls

# should use a single fetch when requesting relationship subfields and provided keys from the same service
query {
    topReviews {
        body
        author {
            id
            reviews {
                body
            }
        }
    }
}
