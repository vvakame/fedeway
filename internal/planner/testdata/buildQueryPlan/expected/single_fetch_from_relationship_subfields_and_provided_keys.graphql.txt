QueryPlan {
	Fetch(service: "reviews") {
		query {
			topReviews {
				body
				author {
					id
					reviews {
						body
					}
				}
			}
		}
	},
}